src/ui/Tutorials/TutorialsSections/SmartContractsSection.jsx

```diff
@@ -21,13 +21,13 @@ export const SmartContractsSection = () => (
     </Text>
 
     <Text>
-      Exchange Contract - 0x00…. - Etherscan link <br/>
+      Exchange Contract - 0x5d22045DAcEAB03B158031eCB7D9d06Fad24609b <br/>
     </Text>
   </SubSection>
   <SubSection id="RopstenContracts" className="section">
     <SubTitle>Ropsten Contracts</SubTitle>
     <Text>
-      Exchange Contract - 0x00…. - Etherscan link <br/>
+      Exchange Contract - 0x9ae0A2EEa2Fc13475C4026f65F37d91Dce45c081 <br/>
```

Comment:
For the ropsten exchange contract it should be 0x5783323064dDa4A1ebe62defFeF46750BD2C560c

Reply:
changed